addons/hr_holidays/report/hr_leave_report_calendar.py

```diff
@@ -38,7 +40,8 @@ def init(self):
             hl.date_to AS stop_datetime,
             hl.employee_id AS employee_id,
             hl.state AS state,
-            em.company_id AS company_id
+            em.company_id AS company_id,
+            state = ANY('{"confirm", "validate1"}') as is_hatched
```

Comment:
It could be simplified to 
```sql
state != 'validate' as is_hatched
```

as we are filtering on `confirm / validate / validate1`

Reply:
Done